tests/test_monitors.py

```diff
@@ -0,0 +1,184 @@
+from scrapy import Spider
+from scrapy.crawler import Crawler
+from spidermon.contrib.scrapy.runners import SpiderMonitorRunner
+from spidermon.contrib.scrapy.monitors import (
+    FinishReasonMonitor, ItemCountMonitor, LogMonitor, UnwantedHttpStatus,
+    SPIDERMON_MIN_ITEMS, SPIDERMON_EXPECTED_FINISH_REASONS,
+    SPIDERMON_MAX_ERROR, SPIDERMON_MAX_UNWANTED_HTTP_CODES)
+from spidermon import MonitorSuite
+
+
+def new_crawler():
+    return Crawler(Spider)
+
+
+def new_suite(monitors):
+    return MonitorSuite(monitors=monitors)
+
+
+def test_needs_to_configure_item_count_monitor():
```

Comment:
What do you think of wrapping these tests in a `TestCase`?
This would make it possible add a `setUp` method to instantiate the crawler, spider and all other studd that is repeated among some of the tests.

Reply:
We use pytest, so it is not necessary to create a `TestCase`. 
As all tests creates a crawler and a spider, you could use [setup_function](https://docs.pytest.org/en/latest/xunit_setup.html?highlight=setup_function) function that will be executed before each function.